lib/posix-process/exportsyms.uk

```diff
@@ -0,0 +1,86 @@
+execl
+execle
+execlp
+execv
+execve
+uk_syscall_r_execve
+uk_syscall_e_execve
+execvp
+execvpe
+getpgid
+uk_syscall_r_getpgid
+uk_syscall_e_getpgid
+getpgrp
+uk_syscall_r_getpgrp
+getpriority
+uk_syscall_r_getpriority
+uk_syscall_e_getpriority
+getrlimit
+uk_syscall_r_getrlimit
+uk_syscall_e_getrlimit
+getrusage
+uk_syscall_r_getrusage
+uk_syscall_e_getrusage
+getsid
+uk_syscall_r_getsid
+uk_syscall_e_getsid
+nice
+pclose
+popen
+prctl
+uk_syscall_r_prctl
+uk_syscall_e_prctl
+prlimit
+uk_syscall_r_prlimit64
+uk_syscall_e_prlimit64
+setpgid
+uk_syscall_r_setpgid
+uk_syscall_e_setpgid
+setpgrp
+uk_syscall_e_getpgrp
+setpriority
+uk_syscall_r_setpriority
+uk_syscall_e_setpriority
+setrlimit
+uk_syscall_r_setrlimit
+uk_syscall_e_setrlimit
+setsid
+uk_syscall_r_setsid
+uk_syscall_e_setsid
+system
+tcgetpgrp
+tcsetpgrp
+wait
+wait3
+wait4
+uk_syscall_r_wait4
+uk_syscall_e_wait4
+waitpid
+waitid
+uk_syscall_r_waitid
+uk_syscall_e_waitid
+getpid
+uk_syscall_r_getpid
+uk_syscall_e_getpid
+getppid
+uk_syscall_r_getppid
+uk_syscall_e_getppid
+gettid
+uk_syscall_r_gettid
+uk_syscall_e_gettid
+exit
+uk_syscall_r_exit
+uk_syscall_e_exit
+exit_group
+uk_syscall_r_exit_group
+uk_syscall_e_exit_group
+uk_posix_process_create
+uk_posix_process_kill
+clone
+uk_syscall_r_clone
+uk_syscall_e_clone
+uk_syscall_r_clone3
+uk_syscall_e_clone3
+# PLEASE NOTE THIS IS NOT A REAL FORK
```

Comment:
Can you also say that they are used for testing and refer to the test that uses them?